include/SFML/System/Clock.hpp

```diff
@@ -107,7 +108,47 @@ class SFML_SYSTEM_API Clock
     /// \return Time elapsed
     ///
     ////////////////////////////////////////////////////////////
-    Time getElapsedTime() const;
+    [[nodiscard]] Time getElapsedTime() const noexcept;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Check whether certain time has passed
+    ///
+    /// This function returns boolean value based on whether certain time has passed.
+    ///
+    /// \return True whether certain time has passed, false otherwise
+    ///
+    ////////////////////////////////////////////////////////////
+    [[nodiscard]] bool hasTimePassed(const Time& time) const noexcept;
```

Comment:
Is this equivalent to `clock.getElapsedTime() > time`?

Reply:
It is equivalent. Motivation was just to have dedicated method to perform these checks. I understand some may find this addition redundant. Also I can see your point with this method being less versetile than comparision operators which sf::Time has plenty.